packages/sms-46elks/plugin.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-console */
+import { Plugin, setupProvider } from '@zemble/core'
+import yoga from '@zemble/graphql'
+
+import type { IStandardSendSmsService } from '@zemble/core'
+
+interface Sms46ElksConfig extends Zemble.GlobalConfig {
+  readonly ELKS_USERNAME?: string
+  readonly ELKS_PASSWORD?: string
+  readonly disable?: boolean
+  readonly options: {
+    readonly dryrun?: 'yes' | 'no'
+    readonly flashsms?: 'yes' | 'no'
+    readonly dontlog?: 'message' | 'from' | 'to'
+    readonly whendelivered?: string
```

Comment:
Might be more of a v2 thing, but since whendelivered is a webhook URL we could set up a route for receiving it (with the routes package), so the developer using the plugin doesn't need to :)